src/nvim/os/win_defs.h

```diff
@@ -21,12 +21,14 @@
 // - SYS_VIMRC_FILE
 // - SPECIAL_WILDCHAR
 
-// _access(): https://msdn.microsoft.com/en-us/library/1w06ktdy.aspx
-#ifndef R_OK
-# define R_OK 4
-#endif
-#ifndef W_OK
-# define W_OK 2
+typedef SSIZE_T ssize_t;
+
+#ifndef SSIZE_MAX
+# ifdef _WIN64
+#  define SSIZE_MAX _I64_MAX
+# else
+#  define SSIZE_MAX LONG_MAX
+# endif
```

Comment:
:+1:  Going on your comment at https://github.com/neovim/neovim/issues/1749#issuecomment-72323402 and the fact that I don't see any contrary suggestions.